packages/frontend/core/src/modules/app-sidebar/views/app-download-button/index.tsx

```diff
@@ -14,10 +14,16 @@ export function AppDownloadButton({
   className?: string;
   style?: React.CSSProperties;
 }) {
-  const [show, setShow] = useState(true);
+  const [show, setShow] = useState(null);
+
+  useEffect(() => {
+    const isClosed = localStorage.getItem('appDownloadClosed') === 'true';
+    setShow(!isClosed);
+  }, []);
 
   const handleClose = useCatchEventCallback(() => {
```

Comment:
@hwangdev97 do we still want to always show the download desktop app button on web?
also the implementation here may follow dismissChangelog in `useAppUpdater`